### PR TITLE
Added another website

### PR DESCRIPTION
**Pull request by CBID2**
I could not find the icon for Dev.to so I used the FaFev one. If you can help me add the correct icon, that'd be great! :)

## Check List (Check all the applicable boxes) 

- [X ] My code follows the code style of this project.
- [ X] My change requires changes to the documentation.
- [ X] I have updated the documentation accordingly.
- [X ] All new and existing tests passed.
- [ X] This PR does not contain plagiarized content.
- [ X] The title of my pull request is a short description of the requested changes.

## Note to reviewers

I could not find the icon for Dev.to, so I used the FaFev one. Can help me add the Dev.to one please?

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/3732"></a>

### CODE REVIEW

**Comment by loftwah** [comment]
The JSON isn't valid.

Try this:

```json
{
	"name": "Christine Belzie",
	"displayStatsPublic": true,
	"type": "personal",
	"bio": "An Instructional Designer who codes",
	"avatar": "https://github.com/CBID2.png",
	"links": [{
			"name": "Follow me on GitHub",
			"url": "https://github.com/CBID2",
			"icon": "FaGithub"
		},
		{
			"name": "Follow me on LinkedIn",
			"url": "https://www.linkedin.com/in/christinebelzie",
			"icon": "FaLinkedin"
		},
		{
			"name": "Follow my tech blog",
			"url": "https://chrissycodes.hashnode.dev/",
			"icon": "SiHashnode"
		},
		{
			"name": "Follow me on Dev.to",
			"url": "https://dev.to/cbid2",
			"icon": "FaFev"
		}
	],
	"milestones": [{
			"title": "Earned a Pull Shark",
			"date": "October 2022",
			"icon": "FaGithub",
			"color": "grey",
			"description": "My first award on GitHub",
			"url": "https://github.com/users/CBID2/achievements/pull-shark"
		},
		{
			"title": "SheSharp New in Tech Scholarship Receipient",
			"date": "November 2022",
			"icon": "FaGithub",
			"color": "grey",
			"description": "Won for being an active participant in SheSharp.",
			"url": "https://www.linkedin.com/posts/christinebelzie_announcing-our-new-in-tech-scholarship-winner-activity-6993952117304508416-3oC5"
		},
		{
			"title": "Wrote my first blog post",
			"date": "December 2022",
			"icon": "SiHashnode",
			"color": "grey",
			"description": "In this blog post, I describe why I decided to learn to code and how I overcame imposter syndrome to start this journey",
			"url": "https://chrissycodes.hashnode.dev/"
		}
	],
	"testimonials": ["loftwah"]
}
```

File: data/CBID2.json [hunk 1]
```diff
@@ -21,6 +21,11 @@
       "icon": "SiHashnode"
     }
   ],
+  {
+  "name": "Follow me on Dev.to",
+  "url": "https://dev.to/cbid2",
+  "icon": "FaFev"
+  },
```
Comment (naomi-lgbt) [inline data/CBID2.json:28]:
```suggestion
    {
    "name": "Follow me on Dev.to",
    "url": "https://dev.to/cbid2",
    "icon": "FaFev"
    }
  ],
```

As a committable suggestion

**Review by stevenovak123 — APPROVED** [review]
LGTM

**Comment by CBID2** [comment]
> LGTM

Yay! :)

**Review by naomi-lgbt — APPROVED** [review]
banger pr, thanks for contributing!

**Pull request merged**